build/cibuild.cmd

```diff
@@ -1,3 +1,3 @@
 @echo off
-powershell -NoLogo -NoProfile -ExecutionPolicy ByPass "& '%~dp0build.ps1'" -build -pack -sign -ci -prepareMachine %*
```

Comment:
Should we really drop `-sign`? IIRC it only validates that the official build would sign things, when in CI mode.

Reply:
Would we catch missing entries in `signtooldata.json` if we take it out?

Reply:
No, I don't think we would--that's why I'm asking if it should be put back in.

Reply:
Oh, I only now realize I actually deleted `-sign`. I'll go ahead and add it back :)